app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.count(" ") == 2:
+        command, old_file, new_file = command.split()
```

Comment:
rename old_file, new_file
old_file -> source_file_path
new_file -> destination_file_path

Reply:
checking number of command's parts was  requested by previous mentor